### ISSUE TITLE
Common challenging concepts or misconceptions

### ISSUE THREAD

**Issue by ChristinaLK** [carpentries-incubator/docker-introduction#56]
Once #55 is merged, please consider adding common misconceptions or challenging threshold concepts to the the instructor notes or this issue.

**Comment by jsgro** [comment]
There are multiple layers of "difficulties" that are not yet mentioned in the lesson but might be too much for this course at this time. These ideas come from my self-teaching Docker last year to implement my tutorial https://bcrf.biochem.wisc.edu/docker-beginner-for-biologists/

Namely: difficulties raised by docker that would run graphical software:

- How to use an X11 program
- How to map and use a Web serving address (useful for RStudio)

There are examples and exercises on the 4th lesson here:
https://static-bcrf.biochem.wisc.edu/tutorials/docker/Docker_04.html

I only tested those on Mac, but with Xwin32 or MobaXterm it might be possible to run X11 on Windows as well... But this is certainly complicated.

**Comment by jsgro** [comment]
Another challenge is about shared directories, at least it's a new one for new Docker 3.0.0 that I updated today from 2.5.x... At least on a Mac, it is not possible for me today to share a folder that is on the Desktop e.g. with `$(pwd):/test` even if I add this location in the Preferences > Resources > File Sharing. Solution was to move the folder within my /Users/jsgro home folder and then it worked.

**Comment by ChristinaLK** [comment]
Thanks for contributing these ideas @jsgro ! 
I'm going to move these comments to a separate issue, because I want to keep this thread focused on learner *misconceptions* or *confusion* about docker, not necessarily technical challenges with running it.
